Example/PaymentSheet Example/PaymentSheet Example/ExampleCustomDeferredCheckoutViewController.swift

```diff
@@ -0,0 +1,334 @@
+            }
+        }
+    }
+
+    @IBAction func hotDogStepperDidChange() {
+        updateUI()
+    }
+
+    @IBAction func saladStepperDidChange() {
+        updateUI()
+    }
+
+    @IBAction func subscribeSwitchDidChange() {
+        updateUI()
+    }
+
+    // MARK: - Helper methods
+
+    private func updateUI() {
+        // Disable buy and payment method buttons
+        buyButton.isEnabled = false
+        paymentMethodButton.isEnabled = false
+
+        fetchTotals { [weak self] in
+            guard let self = self else { return }
+            self.updateLabels()
+
+            // Update PaymentSheet with the latest `intentConfig`
+            self.paymentSheetFlowController.update(intentConfiguration: self.intentConfig) { error  in
+                if error != nil {
+                    // Retry
+                    self.updateUI()
+                } else {
+                    // Re-enable your "Buy" and "Payment method" buttons
+                    self.updateButtons()
+                  }
+            }
+        }
+    }
+
+    func updateButtons() {
+        paymentMethodButton.isEnabled = true
+
+        // MARK: Update the payment method and buy buttons
+        if let paymentOption = paymentSheetFlowController.paymentOption {
+            paymentMethodButton.setTitle(paymentOption.label, for: .normal)
+            paymentMethodButton.setTitleColor(.black, for: .normal)
+            paymentMethodImage.image = paymentOption.image
+            buyButton.isEnabled = true
+        } else {
+            paymentMethodButton.setTitle("Select", for: .normal)
+            paymentMethodButton.setTitleColor(.systemBlue, for: .normal)
+            paymentMethodImage.image = nil
+            buyButton.isEnabled = false
+        }
+    }
+
+    func updateLabels() {
+        hotDogQuantityLabel.text = "\(Int(hotDogStepper.value))"
+        saladQuantityLabel.text = "\(Int(saladStepper.value))"
+
+        subtotalLabel.text = "\(currencyFormatter.string(from: NSNumber(value: computedTotals.subtotal)) ?? "")"
+        salesTaxLabel.text = "\(currencyFormatter.string(from: NSNumber(value: computedTotals.tax)) ?? "")"
+        totalLabel.text = "\(currencyFormatter.string(from: NSNumber(value: computedTotals.total)) ?? "")"
+    }
+
+    func displayAlert(_ message: String) {
+        let alertController = UIAlertController(title: "", message: message, preferredStyle: .alert)
+        let OKAction = UIAlertAction(title: "OK", style: .default) { (_) in
+            alertController.dismiss(animated: true) {
+                self.navigationController?.popViewController(animated: true)
+            }
+        }
+        alertController.addAction(OKAction)
+        present(alertController, animated: true, completion: nil)
+    }
+
+    // MARK: Confirm handler
+
+    func confirmHandler(_ paymentMethodID: String,
+                        _ intentCreationCallback: @escaping (Result<String, Error>) -> Void) {
```

Comment:
let's show the server-side confirmation example

Reply:
Switched!